src/corehost/cli/test/mockcoreclr/mockcoreclr.cpp

```diff
@@ -0,0 +1,209 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "mockcoreclr.h"
+#include <iostream>
+#include "trace.h"
+
+extern "C" pal::hresult_t STDMETHODCALLTYPE coreclr_initialize(
+    const char* exePath,
+    const char* appDomainFriendlyName,
+    int propertyCount,
+    const char** propertyKeys,
+    const char** propertyValues,
+    coreclr_t::host_handle_t* hostHandle,
+    unsigned int* domainId)
+{
+    std::cout << "MockCoreClr::coreclr_initialize(" <<
+        "exePath: " << exePath << ", " <<
```

Comment:
Nit: I found it handy to prefix all the lines with "mock" - it makes it easier to consume from the tests since it won't collide with anything else (some of the names are rather generic).